tests/rptest/services/openmessaging_benchmark.py

```diff
@@ -0,0 +1,228 @@
+    STDOUT_STDERR_CAPTURE = os.path.join(PERSISTENT_ROOT, "becnhmark.log")
+    OPENMESSAGING_DIR = "/opt/openmessaging-benchmark"
+    DRIVER_FILE = os.path.join(OPENMESSAGING_DIR,
+                               "driver-redpanda/redpanda-ducktape.yaml")
+    WORKLOAD_FILE = os.path.join(OPENMESSAGING_DIR, "workloads/ducktape.yaml")
+    OUTPUT_FILE = os.path.join(PERSISTENT_ROOT, "result.json")
+
+    logs = {
+        "open_messaging_benchmark_stdout_stderr": {
+            "path": STDOUT_STDERR_CAPTURE,
+            "collect_default": True
+        },
+        "open_messaging_benchmark_result": {
+            "path": OUTPUT_FILE,
+            "collect_default": True
+        }
+    }
+
+    def __init__(self, ctx, redpanda):
+        super(OpenMessagingBenchmark, self).__init__(ctx, num_nodes=1)
+        self._ctx = ctx
+        self.redpanda = redpanda
+        self.set_default_configuration()
+
+    def set_default_configuration(self):
+        self.configuration = {
+            "topics": 1,
+            "partitions_per_topic": 3,
+            "subscriptions_per_topic": 1,
+            "producers_per_topic": 1,
+            "consumer_per_subscription": 1,
+            "consumer_backlog_size_GB": 0,
+        }
+        if os.environ.get('CI', None) == 'true':
+            self.configuration["producer_rate"] = 10
+            self.configuration["test_duration_minutes"] = 5
+            self.configuration["warmup_duration_minutes"] = 5
+        else:
+            self.configuration["producer_rate"] = 10
+            self.configuration["test_duration_minutes"] = 1
+            self.configuration["warmup_duration_minutes"] = 1
+
+    def set_configuration(self, config):
+        for key, value in config.items():
+            self.configuration[key] = value
+
+    def _create_benchmark_workload_file(self, node):
+        conf = self.render("omb_workload.yaml", **self.configuration)
+
+        self.logger.debug(
+            "Open Messaging Bechmark: Workload configuration: \n %s", conf)
+        node.account.create_file(OpenMessagingBenchmark.WORKLOAD_FILE, conf)
+
+    def _create_benchmark_driver_file(self, node):
+        rp_node = self.redpanda.nodes[0].account.hostname
+        conf = self.render("omb_driver.yaml", redpanda_node=rp_node)
+
+        self.logger.debug(
+            "Open Messaging Bechmark: Driver configuration: \n %s", conf)
+        node.account.create_file(OpenMessagingBenchmark.DRIVER_FILE, conf)
+
+    def _create_workers(self):
+        consumer_workers_amount = self.configuration[
+            "topics"] * self.configuration[
+                "subscriptions_per_topic"] * self.configuration[
+                    "consumer_per_subscription"]
+        producer_workers_amount = self.configuration[
+            "topics"] * self.configuration["producers_per_topic"]
+        workers_amount = consumer_workers_amount + producer_workers_amount
+        self.workers = OpenMessagingBenchmarkWorker(self._ctx,
```

Comment:
nit:The name "OpenMessagingBenchmarkWorker" reads as if a single worker is created instead of multiple workers.

Reply:
Agree. Perhaps we should rename `OpenMessagingBenchmarkWorker` to `OpenMessagingBenchmarkWorkers`?